source/_app/helpers/utils.js

```diff
@@ -70,6 +70,13 @@ app.utils = {
   isNumeric: function(string) {
     var hasNumber = /^\d+$/;
     return hasNumber.test(string);
-  }
+  },
+
+  /**
+   * Pad out numbers < 10
+   */
+  pad: function(n) {
+    return (n < 10) ? ('0' + n) : n;
```

Comment:
Looks like this takes a number, and returns a number if `n` is less than 10, or returns a string if not. This may not matter, depends how it's used, I'm flagging for verification that this won't have unexpected consequences.